operations/environments/pr/main.tf

```diff
@@ -43,4 +43,5 @@ module "template" {
   deployer_id = "d59c2c86-de5e-41b7-a752-0869a73f5a60" //github app registration in Flexion Azure Entra
 
   depends_on = [azurerm_resource_group.group, azurerm_virtual_network.vnet]
+  cron       = "0 30 9 * Feb Mon"
```

Comment:
Is this supposed to run only in February?

Reply:
We have a separate task to update the cron once we have a few more answers from CA. This makes it not-run-yet, which is fine for now

Reply:
More specifically we split this out to separate so that all of the environments don't hit the CA SFTP server all at the same time